### DISCUSSION TITLE
Wallet Connect V2 Update

### DISCUSSION THREAD

**Discussion by a-znk** [rainbow-me/rainbowkit#1454] [Q&A] [answered]
Pardon my ignorance, but do I simply need a project ID to migrate, or do I need to create and complete the full project.

I just want RainbowKit to worth with my dApp, so I don't need much here...

Thanks!

**Comment by DanielSinclair** [comment, marked as answer]
Yes, you'll need a unique `projectId` obtained from [WalletConnect Cloud](https://cloud.walletconnect.com/). You will also need to ensure that you have properly whitelisted your domain through their dashboard and published your dApp project there. Otherwise some of your users will see throttling and failures in WalletConnect session pairing. This is unfortunately a requirement of the WalletConnect v2 protocol, and not RainbowKit.

**Reply by a-znk** [reply]
Thank you!
